fe/src/main/java/org/apache/doris/transaction/GlobalTransactionMgr.java

```diff
@@ -448,35 +454,40 @@ public void abortTransaction(Long dbId, String label, String reason) throws User
                 continue;
             }
             db.readLock();
-            writeLock();
             try {
```

Comment:
why not use write lock

Reply:
There is no changing here, so read lock is enough.